.github/workflows/language-reference.yaml

```diff
@@ -30,22 +30,14 @@ jobs:
           fetch-depth: 0
           ssh-key: ${{ secrets.DOCS_KEY }}
 
-      - name: Cache Coursier
-        uses: actions/cache@v1
-        with:
-          path: ~/.cache/coursier
-          key: sbt-coursier-cache
-
-      - name: Cache SBT
-        uses: actions/cache@v1
-        with:
-          path: ~/.sbt
-          key: sbt-${{ hashFiles('**/build.sbt') }}
+      - name: Cache coursier and sbt
+      - uses: coursier/cache-action@v6
 
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
         with:
-          java-version: 11
+          distribution: 'temurin'
+          java-version: 17
```

Comment:
Here and in a couple other places I sort of made an assumption that 17 shouldn't be an issue at all. If this isn't the case, please let me know and I can switch it back.